pulser-core/pulser/sequence/sequence.py

```diff
@@ -615,9 +620,7 @@ def _config_detuning_map(
         # Manually store the channel declaration as a regular call
         if to_store:
```

Comment:
Do you still need the `to_store` argument? Can't we just put the store in `config_detuning_map` and not in `_config_detuning_map`?